androidsdk/src/main/java/com/wootric/androidsdk/views/phone/SurveyLayoutPhone.java

```diff
@@ -336,7 +332,6 @@ private void setColors() {
         mColorEnabled = res.getColor(mSettings.getSurveyColor());
         mRatingBar.setSelectedColor(mColorSelected);
 
-        mBtnEditScore.setTextColor(mColorEnabled);
```

Comment:
Yeah, you would need to do something like that since the text color will be white now always, what will change is the background.